tools/cloud/download.xml

```diff
@@ -26,7 +26,7 @@
         <param name="overwrite_existing" type="boolean" label="overwrite_existing" />
     </inputs>
     <outputs>
-        <data name="output" type="data" label="Output log" hidden="false"/>
+        <data name="output" type="data" label="Download to cloud log" hidden="false"/>
```

Comment:
I saw the commit message and got excited, but alas -- we're still downloading to the cloud.  What can I do to persuade you to rename this from "Download to cloud" to something like "Send to cloud"?  As we all talked about at the last meeting, "download" has a strong meaning for most people that involves the device they're currently on -- not cloud storage.

Reply:
'Transfer' is a word that has no direction implication and may be useful here.

Reply:
@dannon refactored to `Send`. Do you have any other comments/concerns about this PR?

Reply:
ping @jgoecks ^ (I think you had similar naming concerns).